src/zenml/artifact_stores/base_artifact_store.py

```diff
@@ -475,6 +475,17 @@ def _register(self) -> None:
 
         default_filesystem_registry.register(filesystem_class)
 
+    def _keep_only_latest_file_version(self, path: PathType) -> None:
```

Comment:
Maybe calling this `remove_previous_file_versions(...)` would make it a little clearer? I was confused at first and thought this would return a boolean which indicates whether only the latest file version should be kept.

Reply:
renamed